content/docs/1.6.0/advanced-resources/os-distro-specific/okd-support.md

```diff
@@ -5,6 +5,8 @@ weight: 4
 
 To deploy Longhorn on a cluster provisioned with OpenShift 4.x, some additional configurations are required.
 
+**NOTE**: OKD does not support ARM platform yet. [[OKD website](https://www.okd.io/crc/?h=arm), [OKD in ARM platform github issue](https://github.com/okd-project/okd/issues/1165)]
```

Comment:
```suggestion
> **Note**: OKD currently does not support the ARM platform. For more information, see the [OKD website](https://www.okd.io/crc/?h=arm) and [GitHub issue #1165](https://github.com/okd-project/okd/issues/1165) (*OKD in ARM platform*).
```

Reply:
Modified for all versions after v1.6.0. Thanks.